Pinta.Core/Effects/Utility.cs

```diff
@@ -19,68 +20,62 @@ internal static bool IsNumber (float x)
 
 	public static double Clamp (double x, double min, double max)
 	{
-		if (x < min) {
+		if (x < min)
```

Comment:
Can these first three `Clamp` methods just be replaced by `System.Math.Clamp()`?

Reply:
Sure. It seems to do the same job. I removed it and replaced all of its occurrences by `System.Math.Clamp()`

Reply:
Looks good, thanks!
One further thought is that the ClampToByte methods could be reimplemented as just `(byte) Math.Clamp(x, 0, 255)`?

Reply:
No problem. Done